docs/user-guides/finetuner.md

```diff
@@ -174,10 +177,16 @@ executors:
     replicas: 1
 ```
 
+
 ```{warning}
-Note that Finetuner only support ViT-B/32 CLIP model currently. The model name should match the fine-tuned model, or you will get incorrect output.
+Note that `finetuner`==0.6.2 doesn't support these new clip models trained on Laion2B: 
+- ViT-B-32::laion2b-s34b-b79k
+- ViT-L-14::laion2b-s32b-b82k
+- ViT-H-14::laion2b-s32b-b79k
+- ViT-g-14::laion2b-s12b-b42k
 ```
```

Comment:
Can we provide the full model list supported in finetuner. This is not a release notes where the updated information is fine. As the official documentation, we should provide complete information.

Reply:
Model names in finetuner are not the same as in cas, which is confusing. Should we creat a mapping for the model names?